include/pgettext.php

```diff
@@ -44,7 +44,7 @@ function get_browser_language() {
 	// check if we have translations for the preferred languages and pick the 1st that has
 	for ($i=0; $i<count($lang_list); $i++) {
 		$lang = $lang_list[$i];
-		if(file_exists("view/lang/$lang") && is_dir("view/lang/$lang")) {
+		if($lang === 'en' || file_exists("view/lang/$lang") && is_dir("view/lang/$lang")) {
```

Comment:
Yes, since @Quix0r is supposed to do a whole pass for all existing files, I'm only picky on new files.